sdk/browser/test/integration/otp/index.test.ts

```diff
@@ -39,174 +29,154 @@ const credentialShareRequestToken =
   '50b739ac0e5eb4add1961c88d9f0486b37be928bccf2b19fb5a1d2b7c9bbe'
 
 const options: __dangerous.SdkOptions = getOptionsForEnvironment()
+const { env } = options
 
-describe('AffinityWallet (flows that require OTP)', () => {
-  it('Save Open Attestation credential and #deleteCredential scenario', async () => {
-    const cognitoUsername = generateEmail()
+const messageParameters: MessageParameters = {
+  message: `Your verification code is: {{CODE}}`,
+  subject: `Verification code`,
+}
 
-    const token = await AffinityWallet.signUp(cognitoUsername, cognitoPassword, options)
+const waitForOtpCode = async (inbox: __dangerous.TestmailInbox): Promise<string> => {
```

Comment:
Why is it in __dangerous part of the main code rather than in tests?

@DenisPopov15 , can we move TestmailInbox out into a separate npm package, please?

Reply:
It's an okay-ish start. At least it will be easier to move to a separate package, since there is no copy paste. For a separate package I think there must be a separate task.